### PR TITLE
Add Vietnamese code of conduct

### PR DESCRIPTION
**Pull request by huynguyennovem**
Add Vietnamese code of conduct page to website
Fix [#18597](https://github.com/kubernetes/website/issues/18597)

Signed-off-by: Nguyen Quang Huy huynguyennovem@gmail.com

### CODE REVIEW

File: content/vi/community/code-of-conduct.md [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+Nếu bạn nhận thấy rằng điều này đã lỗi thời, xin vui lòng
+<a href="https://github.com/kubernetes/website/issues/new">gửi một vấn đề</a>.
+
+Nếu bạn nhận thấy vi phạm Quy tắc ứng xử tại một sự kiện hoặc cuộc họp, trong
+Slack, hoặc trong một cơ chế giao tiếp khác, hãy liên lạc với <a href="https://git.k8s.io/community/committee-code-of-conduct">Ủy ban Quy tắc ứng xử Kubernetes</a>. 
+Bạn có thể liên hệ với chúng tôi qua email tại <a href="mailto:conduct@kubernetes.io">conduct@kubernetes.io</a>.
+Sự ẩn danh của bạn sẽ được bảo vệ.
+
+<div class="cncf_coc_container">
+{{< include "/static/cncf-code-of-conduct.md" >}}
```
Comment (kbhawkey) [inline content/vi/community/code-of-conduct.md]: Hello @huynguyennovem . 
Would you look into your PR build failure:
```
0:53:51 PM: hugo --enableGitInfo --buildFuture -b https://deploy-preview-19020--kubernetes-io-master-staging.netlify.com
10:53:51 PM: Building sites …
10:54:13 PM: ERROR 2020/02/07 03:54:13 [vi] no Resource or Page matching "/static/cncf-code-of-conduct.md*".
10:54:13 PM: Total in 22038 ms
```

Comment (truongnh1992) [reply content/vi/community/code-of-conduct.md]: @huynguyennovem please take a look on this.

Comment (huynguyennovem) [reply content/vi/community/code-of-conduct.md]: @kbhawkey I've just added missing files. Thank you for pointing that out.

**Comment by kbhawkey** [comment]
Ping @kubernetes/sig-docs-vi-owners

**Pull request merged**